canonical-web-and-design/ubuntu.com
# Remove Allan Gray from /financial-services

## Done

- Remove Allan Gray from /financial-services

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/financial-services
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to and resolve comments in the [copy doc](https://docs.google.com/document/d/1o9s_MFaiZZIS26tusRi0ALNrcqUihhTSZgxS55oHZj4/edit?ts=5fb695b9#)


## Issue / Card

Fixes #8792
